backends/rapidpro/backend_test.go

```diff
@@ -473,7 +473,7 @@ func (ts *BackendTestSuite) TestMsgStatus() {
 		}
 		err := ts.b.WriteStatusUpdate(ctx, statusObj)
 		ts.NoError(err)
-		time.Sleep(500 * time.Millisecond) // give committer time to write this
```

Comment:
previously the timeout started when the batcher was created.. now it starts whenever a value is queued so we have to give the test a little more leeway to be sure the batcher triggers a batch